tests/test_accessors.py

```diff
@@ -7,39 +7,39 @@
 @pytest.mark.parametrize(
     "geog, expected",
     [
-        (spherely.Point(0, 0), spherely.Point(0, 0)),
+        (spherely.point(0, 0), spherely.point(0, 0)),
         (
-            spherely.LineString([(0, 0), (0, 2)]),
-            spherely.Point(0, 1),
+            spherely.linestring([(0, 0), (2, 0)]),
+            spherely.point(1, 0),
         ),
         (
-            spherely.Polygon([(0, 0), (2, 0), (2, 2), (0, 2)]),
-            spherely.Point(1, 1),
+            spherely.polygon([(0, 0), (0, 2), (2, 2), (2, 0)]),
+            spherely.point(1, 1),
         ),
     ],
 )
 def test_centroid(geog, expected) -> None:
     # scalar
     actual = spherely.centroid(geog)
-    assert isinstance(actual, spherely.Point)
+    assert spherely.get_type_id(actual) == spherely.GeographyType.POINT.value
```

Comment:
```suggestion
    assert spherely.get_type_id(actual) == spherely.GeographyType.POINT
```

It doesn't work without the `.value`? 

I would expect that an int-based enum can be compared to integers directly

Reply:
Hmm, that's unfortunate ..